src/components/DetailsPage.js

```diff
@@ -0,0 +1,51 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router';
+import { fetchCovidDataByCountry } from '../redux/Api';
+import Header from './Header';
+
+const DetailsPage = () => {
+  const { name } = useParams();
+  const [state, setState] = useState(null);
+  useEffect(() => {
+    const getDetails = async () => {
+      const countryDetails = await fetchCovidDataByCountry(name);
+      setState(countryDetails);
+    };
+    getDetails();
+  }, []);
+  return (
+    <>
+      {state ? (
+        <div className="DetailContainer">
+          <div>
+            <Header />
+            <h2 className="Country">{state.country}</h2>
+            <img src={state.countryInfo.flag} alt="country flag" />
+            <h2 className="Name">{state.name}</h2>
+            <h2 className="Name">
+              <span>Population : </span>
+              {state.population.toLocaleString()}
+            </h2>
+            <h2 className="Name">
+              <span>Cases : </span>
+              {state.cases.toLocaleString()}
+            </h2>
+            <h2 className="Name">
+              <span>Active : </span>
+              {state.active.toLocaleString()}
+            </h2>
+            <h2 className="Name">
+              <span>Tests : </span>
+              {state.tests.toLocaleString()}
+            </h2>
+            <h2>{}</h2>
```

Comment:
- [ ] I would recommend you follow the given design. Kindly add the alternating colors and the appropriate spacing between elements as we have in the template.
- [ ] Also ensure this change reflects on the live demo.

  Behance Design | Your Design
  :---:|:---:
  ![debas-orig-detail](https://user-images.githubusercontent.com/36542986/152333175-7e415a8c-4b1d-4ef7-8a78-b901bc3935d9.jpg) | ![debas-own-detail](https://user-images.githubusercontent.com/36542986/152333748-c97ca084-2b82-459c-8372-b19d6eb0d138.jpg)

Reply:
Thank you for the feedback, @TedLivist. I have implemented the changes recommended.

Reply:
well addressed this